### PR TITLE
docs: create CODING_STANDARDS.md and update DEVELOPER.md

### PR DESCRIPTION
**Pull request by kyubisation**

### CODE REVIEW

**Review by DavideMininni-Fincons — COMMENTED** [review]
Looks good, I added just a comment for clarification. 👍

**Review by jeripeierSBB — COMMENTED** [review]
Some minor inputs

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+- **Motivation for or Use Case** - explain what are you trying to do and why the current behavior
+  is a bug for you
+- **Browsers and Operating System** - is this a problem with all browsers?
+- **Reproduce the Error** - provide a live example (using StackBlitz or similar) or a unambiguous set of steps
+- **Screenshots** - Due to the visual nature of @sbb-esta/lyne-components, screenshots can help the team
+  triage issues far more quickly than a text description.
+- **Related Issues** - has a similar issue been reported before?
+- **Suggest a Fix** - if you can't fix the bug yourself, perhaps you can point to what might be
+  causing the problem (line of code or commit)
+
```
[the remark this answers is not in the document]
Comment (kyubisation) [reply CONTRIBUTING.md:52]: No, at least I think they speak for themselves. Or would you add a note?

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**